x-pack/plugins/apm/public/components/app/Home/index.tsx

```diff
@@ -84,12 +86,20 @@ interface Props {
 
 export function Home({ tab }: Props) {
   const { config, core } = useApmPluginContext();
-  const canAccessML = !!core.application.capabilities.ml?.canAccessML;
+  const capabilities = core.application.capabilities;
+  const canAccessML = !!capabilities.ml?.canAccessML;
   const homeTabs = getHomeTabs(config);
   const selectedTab = homeTabs.find(
     (homeTab) => homeTab.name === tab
   ) as $ElementType<typeof homeTabs, number>;
 
+  const {
+    isAlertingAvailable,
+    canReadAlerts,
+    canSaveAlerts,
+    canReadAnomalies,
+  } = useAlertingIntegrations();
```

Comment:
I think it's better if this is just an ordinary function, and should be called something more accurate. "Integrations" could be anything really.

```suggestion
  } = getAlertingCapabilities(capabilities);
```